include/aikido/planner/vectorfield/MetaSkeletonStateSpaceVectorFieldIntegrator.hpp

```diff
@@ -0,0 +1,67 @@
+#ifndef AIKIDO_PLANNER_VECTORFIELD_METASKELETONSTATESPACEVECTORFIELDINTEGRATOR_HPP_
+#define AIKIDO_PLANNER_VECTORFIELD_METASKELETONSTATESPACEVECTORFIELDINTEGRATOR_HPP_
+
+#include <functional>
+#include <dart/common/Timer.hpp>
+#include <aikido/constraint/Testable.hpp>
+#include <aikido/planner/vectorfield/MetaSkeletonStateSpaceVectorField.hpp>
+#include <aikido/planner/vectorfield/VectorFieldIntegrator.hpp>
+#include <aikido/planner/vectorfield/VectorFieldPlannerStatus.hpp>
+#include <aikido/planner/vectorfield/VectorFieldUtil.hpp>
+#include <aikido/statespace/dart/MetaSkeletonStateSpace.hpp>
+#include <aikido/trajectory/Spline.hpp>
+
+namespace aikido {
+namespace planner {
+namespace vectorfield {
+
+/// VectorField Planner generates a trajectory by following a vector field
+/// defined in joint space.
+/// A planned trajectroy ends either by a predefined termination criterion or a
+/// integral time.
+///
+/// This class defines two callback functions for a integrator.
+/// step() provides joint velocities the vector field planner should follow,
+/// check() is called after each integration step to check planner status.
+class MetaSkeletonStateSpaceVectorFieldIntegrator : public VectorFieldIntegrator
```

Comment:
I am not sure why this class is necessary. I think we can safely delete this once we implement `convertStateToPositions()` and `convertPositionsToState()` in the base `VectorField` class.